util/kubectl_test.go

```diff
@@ -0,0 +1,10 @@
+package util
+
+import "testing"
+
+func TestCheckKubectl(t *testing.T) {
+	result := CheckKubectl()
```

Comment:
How about making `kubectl` available in our CircleCI build context? Would that be too much?